src/main/java/com/bidahochi/BlockMod/FoxBlocks.java

```diff
@@ -29,9 +33,15 @@ public class FoxBlocks
     public static CreativeTabs foxBlocksCreativeTabProperProps;
     public static CreativeTabs foxBlocksCreativeTabItems;
     public static Logger blockLogger = LogManager.getLogger("FoxBlocks");
+    public static FMLEventChannel channel;
+
+    /* TrainCraft proxy files */
```

Comment:
Foxblocks

Reply:
Also, you should take a look at how Ztones handles it, as while different, it is an example of a scrolling item
https://github.com/GTNewHorizons/Ztones-Unofficial/tree/master/src/main/java/oneeyemaker/ztones

Reply:
I will take a look. Most of my implementation was based around how Carpenter's did it. Having another source to guide will be helpful.